nix/hydra/project.nix

```diff
@@ -21,6 +21,7 @@ let
       # Reason: haskell.nix modules/overlays neds to be last
       # https://github.com/input-output-hk/haskell.nix/issues/1954
       haskellNix.overlay
+      iohk-nix.overlays.haskell-nix-crypto
```

Comment:
@locallycompact Note that I changed this in https://github.com/input-output-hk/hydra/tree/5e38c1ab6c6c4e3f913cfc4d7ee3b308f525b842/

I think the static muslc build would fail if we re-order the overlays this way (see comment).

I also fixed the comments. It seems that `haskell-nix-crypto` is just containing pkg-config reconfiguration via haskell.nix. The libs are in the `crypto` overlay.